config.cson

```diff
@@ -18,3 +18,8 @@
   "advanced-new-file":
     showFilesInAutoComplete: true
     suggestCurrentFilePath: true
+
+# c-sharp settings
```

Comment:
This comment may end up being automatically deleted by Atom, I only discovered this after opening the PR. Should I push a change w/o the comment in place?